control-plane/agents/src/bin/core/controller/reconciler/volume/garbage_collector.rs

```diff
@@ -150,16 +150,16 @@ async fn disown_unused_nexuses(
     context: &PollContext,
 ) -> PollResult {
     let mut results = vec![];
-
     for nexus in context.specs().volume_nexuses(volume.uuid()) {
         if let Ok(mut nexus) = nexus.operation_guard() {
             if let Some(target) = volume.as_ref().target() {
                 if target.nexus() == nexus.uid() || nexus.as_ref().is_shutdown() {
                     continue;
                 }
             }
-
-            nexus.warn_span(|| tracing::warn!("Attempting to disown unused nexus"));
+            nexus.info_span(|| {
+                tracing::info!("Disowning unused nexus from its volume");
```

Comment:
Does the info span append the nexus uuid in it?

Reply:
Yes it does.